react/src/components/ResourceGroupSettingModal.tsx

```diff
@@ -0,0 +1,411 @@
+import { filterEmptyValues } from '../helper';
+import { useCurrentDomainValue } from '../hooks';
+import BAICard from './BAICard';
+import BAIModal from './BAIModal';
+import DomainSelector from './DomainSelector';
+import { ResourceGroupSettingModalAssociateDomainMutation } from './__generated__/ResourceGroupSettingModalAssociateDomainMutation.graphql';
+import { ResourceGroupSettingModalCreateMutation } from './__generated__/ResourceGroupSettingModalCreateMutation.graphql';
+import { ResourceGroupSettingModalFragment$key } from './__generated__/ResourceGroupSettingModalFragment.graphql';
+import { ResourceGroupSettingModalUpdateMutation } from './__generated__/ResourceGroupSettingModalUpdateMutation.graphql';
+import {
+  App,
+  Col,
+  Flex,
+  Form,
+  FormInstance,
+  Input,
+  InputNumber,
+  ModalProps,
+  Row,
+  Select,
+  Skeleton,
+  Switch,
+} from 'antd';
+import { createStyles } from 'antd-style';
+import graphql from 'babel-plugin-relay/macro';
+import _ from 'lodash';
+import { Suspense, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFragment, useMutation } from 'react-relay';
+
+const useStyles = createStyles(({ css }) => ({
+  formItem: css`
+    .ant-form-item-required:before {
+      display: none !important;
```

Comment:
why don't you display required field? too many?

Reply:
Even if Select clear is not allowed, it is better to keep it simple by always displaying the required value if it is mandatory, rather than adding exceptions.